### PR TITLE
Updated typings for "restify-cookies".

### PR DESCRIPTION
**Pull request by weekens**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/nathschmidt/restify-cookies#restify-cookies
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/restify-cookies/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+import * as restify from 'restify';
+
+declare module 'restify' {
+  interface CookieOptions {
+    encode?: (input: string) => string; // tslint:disable-line:prefer-method-signature
+    maxAge?: number;
+    domain?: string;
+    path?: string;
+    expires?: string;
```
Comment (piorek) [inline types/restify-cookies/index.d.ts]: @weekens it should be Date can you fix it?

restify-cookies i using cookie.serialize
https://github.com/nathschmidt/restify-cookies/blob/v0.2.3/index.js#L50

You could use existing cookie definitions from here:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/cookie/index.d.ts#L37

Comment (weekens) [reply types/restify-cookies/index.d.ts]: @piorek Sure, I'll fix that.

I looked at the definitions you provided. In fact, not all fields from that definition are used in restify-cookies (for example, `sameSite` is not used).

Comment (weekens) [reply types/restify-cookies/index.d.ts]: Created new PR: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/20752

Comment (piorek) [reply types/restify-cookies/index.d.ts]: @weekens Thx

**Pull request merged**